### PR TITLE
drop ratio_uncertainty_type argument and replace it with split_ratio comparison argument

### PR DESCRIPTION
**Pull request by cyrraz**
* Drop `ratio_uncertainty_type` argument and replace it with `"split_ratio"` comparison argument.

### CODE REVIEW

File: plothist/comparison.py
Comment (0ctagon) [inline plothist/comparison.py:1]: In the `ValueError`, miss the new comparison in the list.

**Review by 0ctagon — CHANGES_REQUESTED** [review]
Great PR :fire:

File: plothist/plotters.py [hunk 1]
```diff
@@ -609,7 +603,7 @@ def plot_comparison(
             ax.axhline(1, ls="--", lw=1.0, color="black")
             ax.set_ylabel(r"$\frac{" + h1_label + "}{" + h2_label + "}$")
 
-        if ratio_uncertainty_type == "split":
+        if comparison == "split_ratio":
```
[the remark this answers is not in the document]
Comment (cyrraz) [reply plothist/plotters.py:606]: No, as discussed.

File: plothist/plotters.py [hunk 2]
```diff
@@ -591,7 +585,7 @@ def plot_comparison(
         plot_hist_kwargs.setdefault("color", "black")
         plot_error_hist(hist_comparison, ax=ax, **plot_hist_kwargs)
 
-    if comparison in ["ratio", "relative_difference"]:
+    if comparison in ["ratio", "split_ratio", "relative_difference"]:
```
[the remark this answers is not in the document]
Comment (cyrraz) [reply plothist/plotters.py:588]: No, as discussed.

**Review by 0ctagon — APPROVED** [review]

**Pull request merged**